compiler/bindings/python/iree/compiler/tools/import_onnx/__main__.py

```diff
@@ -75,6 +152,34 @@ def parse_arguments(argv=None):
         action="store_true",
         help="Disable verification prior to printing",
     )
+    parser.add_argument(
+        "--data-prop",
+        dest="data_prop",
+        default=True,
+        action=argparse.BooleanOptionalAction,
+        help="Toggle data propogation for onnx shape inference",
+    )
+    parser.add_argument(
+        "--keep-temps", action="store_true", help="Keep intermediate files"
+    )
+    parser.add_argument(
+        "--temp-dir",
+        help="Pre-existing directory in which to create temporary files."
+        ' For example, to place temporaries under the directory "foo/bar",'
+        ' specify --temp-dir=foo/bar.  "foo/bar" must already exist.'
+        " Defaults to the directory of the input file.",
+        type=Path,
+    )
+    parser.add_argument(
+        "--data-dir",
+        help="Path between CWD and the base directory of the data,"
+        " excluding the directories given in the 'location' argument of "
+        " convert_model_to_external_data.  For example, if 'location' was"
+        ' "data/data.bin" and the relative path from CWD to that .bin file is'
+        ' a/b/data/data.bin, then set data-dir to "a/b".'
+        " Defaults to the directory of the input file.",
+        type=Path,
+    )
```

Comment:
This docstring is confusing. Why not just take a regular path?

https://onnx.ai/onnx/api/external_data_helper.html#convert-model-to-external-data takes a relative path, sure, but that's for _creating_ external data files. All we need to do here is load from an external path, which has no requirements on being in the same directory tree as the base model file: https://onnx.ai/onnx/api/external_data_helper.html#load-external-data-for-model

Reply:
Yeah, I don't see anything that suggests this has to be a relative path either. made the change